lib/ansible/modules/cloud/azure/azure_rm_servicebus.py

```diff
@@ -0,0 +1,241 @@
+        except Exception as exc:
+            self.fail("Error creating namespace {0} - {1}".format(self.name, check_name.message or str(check_name)))
+
+    def create(self):
+        self.log('Cannot find namespace, creating a one')
+        try:
+            sku = self.servicebus_models.SBSku(name=str.capitalize(self.sku))
+            poller = self.servicebus_client.namespaces.create_or_update(self.resource_group,
+                                                                        self.name,
+                                                                        self.servicebus_models.SBNamespace(location=self.location,
+                                                                                                           sku=sku))
+            ns = self.get_poller_result(poller)
+        except Exception as exc:
+            self.fail('Error creating namespace {0} - {1}'.format(self.name, str(exc.inner_exception) or str(exc)))
+        return ns
+
+    def delete(self):
+        try:
+            self.servicebus_client.namespaces.delete(self.resource_group, self.name)
+            return True
+        except Exception as exc:
+            self.fail("Error deleting route {0} - {1}".format(self.name, str(exc)))
+
+    def get(self):
+        try:
+            return self.servicebus_client.namespaces.get(self.resource_group, self.name)
+        except Exception:
+            return None
+
+    def to_dict(self, instance):
+        result = dict()
+        attribute_map = self.servicebus_models.SBNamespace._attribute_map
+        for attribute in attribute_map.keys():
+            value = getattr(instance, attribute)
+            if not value:
+                continue
+            if isinstance(value, self.servicebus_models.SBSku):
+                result[attribute] = value.name.lower()
+            elif isinstance(value, datetime):
+                result[attribute] = str(value)
+            elif isinstance(value, str):
+                result[attribute] = to_native(value)
+            elif attribute == 'max_size_in_megabytes':
+                result['max_size_in_mb'] = value
+            else:
+                result[attribute] = value
+        return result
+
+    # SAS policy
+    def get_auth_rules(self):
+        result = dict()
+        try:
+            rules = self.servicebus_client.namespaces.list_authorization_rules(self.resource_group, self.name)
+            while True:
+                rule = rules.next()
+                result[rule.name] = self.policy_to_dict(rule)
+        except Exception:
+            pass
+        return result
+
+    def get_sas_key(self, name):
+        try:
+            return self.servicebus_client.namespaces.list_keys(self.resource_group, self.name, name).as_dict()
+        except Exception:
+            pass
+        return None
+
+    def policy_to_dict(self, rule):
+        result = rule.as_dict()
+        rights = result['rights']
+        if 'Manage' in rights:
```

Comment:
Same as in the sas_policy module; why not just return this as a sorted list instead of making compound values?

Reply:
I make this as a string because these rights have privileged. `Manage` is the admin level can read and write. So there are only four types rights here.